File: Deafios em C#/1012 - Área.cs
using System; 

//Developed by: @LucasMarcuzo

class URI1012 {
    static void Main(string[] args) { 

        string[] values = Console.ReadLine().Split();

        Console.WriteLine("TRIANGULO: " + ((double.Parse(values[0]) * double.Parse(values[2])) /2).ToString("0.000"));
        Console.WriteLine("CIRCULO: " + ((Math.PI*(double.Parse(values[2]) * double.Parse(values[2])))).ToString("0.000"));
        Console.WriteLine("TRAPEZIO: " + (((double.Parse(values[0]) + double.Parse(values[1])) * double.Parse(values[2])) / 2).ToString("0.000"));
        Console.WriteLine("QUADRADO: " + (double.Parse(values[1]) * double.Parse(values[1])).ToString("0.000"));
        Console.WriteLine("RETANGULO: " + ((double.Parse(values[0]) * double.Parse(values[1]))).ToString("0.000"));
        Console.ReadKey();
        
    }
}

File: Desafios em JavaScript/1012 - Área.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

var values = [3];

values[0]=parseFloat(lines.shift());
values[1]=parseFloat(lines.shift());
values[2]=parseFloat(lines.shift());

console.log(`TRIANGULO: ${((values[0] * values[2]) /2).toFixed(3)}`);
console.log(`CIRCULO: ${((3.14159*(values[2] * values[2]))).toFixed(3)}`);
console.log(`TRAPEZIO: ${((((values[0] + values[1])) * values[2]) / 2).toFixed(3)}`);
console.log(`QUADRADO: ${(values[1] * values[1]).toFixed(3)}`);
console.log(`RETANGULO: ${((values[0] * values[1])).toFixed(3)}`); 